src/clients/cassandra/CassandraConnector.js

```diff
@@ -65,13 +62,14 @@ function executeBatchMutations(mutations) {
  */
 function executeQuery(query, params) { // eslint-disable-line no-unused-vars
```

Comment:
@jcjimenez implemented this part, but I can look into paging for this and update it accordingly